lib/openxml/elements/run_properties.rb

```diff
@@ -39,6 +43,10 @@ def initialize(*boolean_properties, **value_properties)
         end
       end
 
+      def font_color=(color)
+        self.push(OpenXml::Elements::RGBColor.new(color))
+      end
```

Comment:
What if I make the methods private so the only way to set them is through the constructor? It does have some drawbacks, but I think they are minimal. You can't reuse a text element with a different color or font would be the drawback. I think this is reasonable.